README.md

```diff
@@ -258,6 +258,48 @@ This object defines templates (functions) that are used for displaying parts of
 
 `suggestion` is a function that receives one argument, a suggestion to be displayed. It is used when rendering suggestions, and should return a string, which can contain HTML. :warning: **Caution:** because this function allows you to output arbitrary HTML, you should [make sure it's trusted](https://en.wikipedia.org/wiki/Cross-site_scripting), and accessible.
 
+### Internationalization
+
+#### `tNoResults` (default: `() => <span>No results found</span>`)
+
+Type: `Function`
+
+A function that receives no arguments and should return the text used in the dropdown to indicate that there are no results.
+
+#### `tStatusQueryTooShort` (default: `(minQueryLength) => <span>Type in {minQueryLength} or more characters for results.</span>`)
+
+Type: `Function`
+
+A function that receives one argument that indicates the minimal amount of characters needed for the dropdown to trigger and should return the text used in the accessibility hint to indicate that the query is too short.
+
+#### `tStatusNoResults` (default: `() => <span>No search results.</span>`)
+
+Type: `Function`
+
+A function that receives no arguments and should return the text that is used in the accessibility hint to indicate that there are no results.
+
+#### `tStatusSelectedOption` (default: `(selectedOption, length) => <span>{selectedOption} (1 of {length}) is selected.</span>`)
+
+Type: `Function`
+
+A function that receives two arguments, the selectedOption and the amount of available options, and it should return the text used in the accessibility hint to indicate which option is selected.
+
+#### `tStatusResults` (default: 
```

Comment:
For this one in particular, because it's more complex, I don't think this works as a heading. Try this:

````md
#### `tStatusResults`

Default:

```js
(length, contentSelectedOption) => { ... }
```

Type: `Function`

A function that...
````